src/http-api.ts

```diff
@@ -33,6 +33,22 @@ export class HttpApi {
   constructor(config: DwnServerConfig, dwn: Dwn, registrationManager?: RegistrationManager) {
     console.log(config);
 
+    this.#packageInfo = {
+      server: config.serverName,
+    };
+    
+    // Server and SDK versions are pulled from `package.json` at runtime.
+    // If running using `npm` the `process.env.npm_package_json` variable exists as the filepath, so we use that.
+    // Otherwise we check to see if a specific `DWN_SERVER_PACKAGE_JSON` exists, if it does we use that.
+    // Finally if both of those options don't exist we resort to the path within the docker server image, located at `/dwn-server/package.json`
```

Comment:
The context info is super useful:

1. Sanity: is it true that `server` is pulled from `package.json` at runtime?
2. Consider moving this to `config.ts`, looks like this class doesn't care about this info.

Reply:
1) Woops! I started that comment before `server` was part of the picture, so I mean the server's version not the `server` text value, but i totally see the confusion! Will update this for better clarity. 

2. will move it there.

Reply:
Also mistakenly removed `process.env.npm_package_name` as the first value to choose from, so added that back in to `config.ts`